Cauchy-Schwarz inequality in complex inner product spaces (#1793)
UniMath/SymmetryBookFormalization: src/complex-numbers/magnitude-complex-numbers.lagda.md

        by ap real-ℝ⁰⁺ (distributive-sqrt-mul-ℝ⁰⁺ _ _)
```

> ### The magnitude of a complex number multiplied on the left by an embedded real number

Reviewer: ```suggestion
### The magnitude of a complex number multiplied on the left by a real number as a complex number
```
